scene/register_scene_types.cpp

```diff
@@ -203,6 +203,7 @@
 #include "scene/main/shader_globals_override.h"
 
 #ifndef _3D_DISABLED
+#include "scene/2d/listener_2d.h"
```

Comment:
Are you sure this should be included in the `#ifndef _3D_DISABLED` block? This is a 2d node so I want to double-check.

Reply:
Rebase mistake, good catch.